fast_api_als/services/verify_phone_and_email.py

```diff
@@ -38,14 +39,22 @@ async def verify_phone_and_email(email: str, phone_number: str) -> bool:
     phone_valid = False
     data = {}
 
+    logging.info("Calling validation service for email and phone")
     await asyncio.gather(
         call_validation_service(email_validation_url, "email", email, data),
         call_validation_service(phone_validation_url, "phone", phone_number, data),
     )
+    logging.info("Call to validation service completed for email and phone")
```

Comment:
does not provide any context to the log viewer

Reply:
Intent behind this log is that time stamp is logged along with this message, just before call and after call, this will help in analysing time taken in request